<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proxier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/assets/list.png</normaloff>:/images/assets/list.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	background: white
}</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>435</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
    background-color:  rgb(66, 69, 74);
}

QTabWidget {
	border: none;
    background-color:  rgb(66, 69, 74);
}

QTabWidget::pane {
    background: red;
}

QTabBar::tab {
	font-family: Impact;
	padding: 3px;
    background-color: white;
	color: rgb(66, 69, 74);
}

QTabBar::tab:selected {
	background-color: rgb(66, 69, 74);
	color: white;
 	margin-bottom: -1px;
}


</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/assets/checker.png</normaloff>:/images/assets/checker.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Checker</string>
     </attribute>
     <widget class="QTableWidget" name="proxies_checker_table">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>23</y>
        <width>531</width>
        <height>321</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    border: 1px solid white;
	border-bottom-right-radius: 10px;
	border-bottom-left-radius: 10px;
	background-image: url(:/images/icon.png);
	background-repeat: no-repeat;
	background-position: center center;
	color: white;
}

</string>
      </property>
      <property name="autoScrollMargin">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>ADDRESS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PORT</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CITY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addlist_checker_btn">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>23</y>
        <width>181</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	border: 1px solid white;
	border-radius: 5px;
	font-family: Impact;
}

QPushButton:hover {
	background: white;
	color: rgb(66, 69, 74);
}

QPushButton::menu-indicator{width:0px;}
</string>
      </property>
      <property name="text">
       <string>PROXY LIST</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/assets/plus.png</normaloff>:/images/assets/plus.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="site_checker_line">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>368</y>
        <width>441</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: white;
	border: 1px solid white;
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
	font-family: Impact;
}</string>
      </property>
      <property name="text">
       <string>http://www.163.com</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_checker_btn">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>63</y>
        <width>181</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	border: 1px solid white;
	border-radius: 5px;
	font-family: Impact;
}

QPushButton:hover {
	background: white;
	color: rgb(66, 69, 74);
}

</string>
      </property>
      <property name="text">
       <string>CLEAR</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/assets/trash.png</normaloff>:/images/assets/trash.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="save_checker_btn">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>103</y>
        <width>181</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	border: 1px solid white;
	border-radius: 5px;
	font-family: Impact;
}

QPushButton:hover {
	background: white;
	color: rgb(66, 69, 74);
}

QPushButton::menu-indicator{width:0px;}
</string>
      </property>
      <property name="text">
       <string>EXPORT</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/assets/save.png</normaloff>:/images/assets/save.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="start_checker_btn">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>314</y>
        <width>181</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	border: 1px solid white;
	border-radius: 5px;
	font-family: Impact;
}

QPushButton:hover {
	background: white;
	color: rgb(66, 69, 74);
}

QPushButton::menu-indicator{width:0px;}
</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/assets/play.PNG</normaloff>:/images/assets/play.PNG</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>37</x>
        <y>163</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;PROXIES:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>183</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;HITS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>223</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;CHECKED:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>203</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;BAD:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>243</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;ERRORS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="proxies_checker_lbl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>164</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="hits_checker_lbl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>184</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#2cff21;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="bad_checker_lbl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>204</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff3c0b;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="errors_checker_lbl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>244</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff3c0b;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="checked_checker_lbl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>224</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>379</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;URL:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>379</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Status: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="source_checker_lbl">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>379</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;nothing to see...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="leecher_tab">
     <attribute name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/assets/leecher.png</normaloff>:/images/assets/leecher.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Leecher</string>
     </attribute>
     <widget class="QTableWidget" name="proxies_fetch_table">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>23</y>
        <width>431</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    border: 1px solid white;
	border-bottom-right-radius: 10px;
	border-bottom-left-radius: 10px;
	background-image: url(:/images/icon.png);
	background-repeat: no-repeat;
	background-position: center center;
	color: white;
}

</string>
      </property>
      <property name="autoScrollMargin">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>ADDRESS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PORT</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SOURCE</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>110</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;LINKS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="link_fetch_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>132</y>
        <width>281</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit {
    border: 1px solid white;
	border-bottom-right-radius: 10px;
	border-bottom-left-radius: 10px;
	background-image: url(:/images/assets/icon (1).png);
	background-repeat: no-repeat;
	background-position: center center;
	color: white;
}</string>
      </property>
      <property name="plainText">
       <string>https://proxy-daily.com/
https://free-proxy-list.net/
http://nntime.com/proxy-list-02.htm
https://www.sslproxies.org/
http://proxy.ipcn.org/proxylist2.html
http://best-proxy.ru/feed
http://www.proxylists.net/?HTTP
http://ab57.ru/downloads/proxyold.txt
http://www.freeproxy.ru/download/lists/goodproxy.txt
http://www.proxylists.net/http_highanon.txt
http://www.atomintersoft.com/high_anonymity_elite_proxy_list
http://www.atomintersoft.com/transparent_proxy_list
http://www.atomintersoft.com/anonymous_proxy_list
http://www.live-socks.net/
http://www.vipsocks24.net/
http://www.proxyserverlist24.top/
http://newfreshproxies-24.blogspot.com
http://www.freshnewproxies24.top/
http://www.socks24.org/
http://www.socksproxylist24.top/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_fetch_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>22</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	border: 1px solid white;
	border-radius: 5px;
	font-family: Impact;
}

QPushButton:hover {
	background: white;
	color: rgb(66, 69, 74);
}</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/assets/play.PNG</normaloff>:/images/assets/play.PNG</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="save_fetch_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>63</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	border: 1px solid white;
	border-radius: 5px;
	font-family: Impact;
}

QPushButton:hover {
	background: white;
	color: rgb(66, 69, 74);
}</string>
      </property>
      <property name="text">
       <string>EXPORT</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/assets/save.png</normaloff>:/images/assets/save.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>342</x>
        <y>384</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Hits:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Status: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="source_fetch_lbl">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>380</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;nothing to see...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="hits_fetch_lbl">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>383</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#2cff21;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>463</x>
        <y>384</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Errors:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="errors_fetch_lbl">
      <property name="geometry">
       <rect>
        <x>513</x>
        <y>383</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff3c0b;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_fetch_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>63</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	border: 1px solid white;
	border-radius: 5px;
	font-family: Impact;
}

QPushButton:hover {
	background: white;
	color: rgb(66, 69, 74);
}</string>
      </property>
      <property name="text">
       <string>CLEAR</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/images/assets/trash.png</normaloff>:/images/assets/trash.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings_tab">
     <attribute name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/assets/settings.png</normaloff>:/images/assets/settings.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Settings</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>435</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="github_btn">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>-1</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/assets/github.jpg</normaloff>:/images/assets/github.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
